network/onchain-discovery/src/lib.rs

```diff
@@ -65,17 +65,17 @@ pub mod types;
 
 /// Query our own storage for the latest discovery set and epoch change proof.
 async fn storage_query_discovery_set(
-    storage_read_client: Arc<dyn StorageRead>,
+    libra_db: Arc<dyn DbReader>,
     req_msg: QueryDiscoverySetRequest,
 ) -> Result<(QueryDiscoverySetRequest, QueryDiscoverySetResponseWithEvent)> {
     let storage_req_msg = Into::<UpdateToLatestLedgerRequest>::into(req_msg.clone());
 
-    let res_msg = storage_read_client
+    let res_msg = libra_db
         .update_to_latest_ledger(
```

Comment:
please don't use this as it's deprecated.

@lightmark  Shall we rename the method so people know not to use it?

Reply:
right, this was just a temporary fix to split this change into two commits. the next commit removes immediately removes this.